### ISSUE TITLE
Using bigint value in a where clause

### ISSUE THREAD

**Issue by mlms13** [jawj/zapatos#172]
I have a record ID of type `bigint`, and I would like to use it in the `where` clause of a Zapatos query, but I'm unable to use it directly:

```ts
async selectOneById(id: bigint): Promise<MyThing> {
  return await db
    .selectExactlyOne(
      'my_cool_table',
      { id: id },
    )
    .run(pool);
  }
```

This produces the type error:

```sh
error TS2322: Type 'bigint' is not assignable to type 'number | `${number}` | SQLFragment<any[], never> | ParentColumn<Column | undefined> | Parameter<number | `${number}`> | SQLFragment<any, number | ... 3 more ... | Parameter<...>> | undefined'.
```

I read through the docs, and I see examples of how to parse bigints _out_ of the result of selecting from the database, but I don't see how to pass bigint values _into_ Zapatos methods. chatgpt suggested doing:

```
`${Number(id)}`
```

But I'm pretty sure this would lead to precision loss and potentially query for the wrong ID.

The best I've come up with so far is basically doing a type conversation that lies about the type:

```ts
const bigIntToNumberTemplate = (value: bigint): `${number}` => `${value}` as `${number}`;
```

I'm wondering if I've missed any better options. Thanks!

**Comment by jawj** [comment]
I see this issue. I'll take a better look as soon as I can.

**Comment by jawj** [comment]
Should be available in 6.4.1. Let me know if not!
